packages/website/src/reduxSlices/files.ts

```diff
@@ -95,6 +95,16 @@ export const selectMapIdToChildren: (state: any) => Record<string, DriveFile[]>
       }
 
       retMap[id].sort((a, b) => {
+        // Show the readme file first
+        if (a.name === 'readme') { return -1 }
```

Comment:
- Maybe you should do a case-insensitive comparing.
- I think we should check `b.name` as well.

How about this:

```js
function getNameOrder(s?: string) {
  if (s?.toLowerCase() === 'readme') {
    return 0;
  } else if (s?.startsWith('.')) {
    return 1;
  } else {
    return 2;
  }
}

const orderA = getNameOrder(a.name);
const orderB = getNameOrder(b.name);
if (orderA < orderB) {
  return -1;
} else if (orderA > orderB) {
  return 1;
} else {
  return naturalCompare(a.name?.toLowerCase() ?? '', b.name?.toLowerCase() ?? '');
}
```

Reply:
I updated the function a bit so that a and b are both checked. I am not really a fan of returning a range of different integers to indicate priority since it is never clear what the priority should be.